ansible/roles/nginx/tasks/main.yml

```diff
@@ -54,6 +54,13 @@
       - install
       - nginx
 
+  - name: Copy Lua Auth Script file
```

Comment:
Might be good to add a remove auth_script.lua task, when nginx_auth_enable is not defined or not true.